requirements/extras/sqs.txt

```diff
@@ -1 +1,4 @@
+boto3>=1.26.143
+pycurl>=7.43.0.5; sys_platform != 'win32' and platform_python_implementation=="CPython"
```

Comment:
is pycurl no longer required for sqs on windows?

Reply:
also why do these 3 need duplicating in here when they're already in https://github.com/celery/kombu/blob/main/requirements/extras/sqs.txt

Reply:
we just don't test it on windows due to lack of wheels